Code_Exercises/Exercise_13_Load_Balancing/source.cpp

```diff
@@ -12,5 +12,31 @@
 #include <catch2/catch.hpp>
 
 TEST_CASE("load_balancing", "load_balancing_source") {
-  REQUIRE(true);
+  constexpr size_t dataSize = 1024;
+  constexpr float ratio = 0.5f;
+  constexpr size_t dataSizeFirst = ratio * dataSize;
+  constexpr size_t dataSizeSecond = dataSize - dataSizeFirst;
+
+  float a[dataSize], b[dataSize], r[dataSize];
+  for (int i = 0; i < dataSize; ++i) {
+    a[i] = static_cast<float>(i);
+    b[i] = static_cast<float>(i);
+    r[i] = 0.0f;
+  }
+
+  // Task: run these two kernels on the SYCL device
```

Comment:
The goal of this exercise is to, if possible, split the work across two distinct devices, we should describe this intent here.

Reply:
I've updated this in 6b653a2